src/strongbox/ui/cli.clj

```diff
@@ -368,6 +368,19 @@
         addon-id (utils/extract-addon-id addon)]
     (add-tab tab-id (or (:dirname addon) (:label addon) (:name addon) "[bug: missing tab name!]") closable? addon-id)))
 
+(defn-spec addon-log-entries ::sp/list-of-maps
+  "returns a list of addon entries for the given `:dirname` since last refresh"
+  [addon map?]
+  (let [not-report #(-> % :level (= :report) not)
+        filter-fn (utils/log-line-filter-with-reports (core/selected-addon-dir) addon)]
+    (->> (core/get-state)
+         :log-lines
+         reverse
+         (filter filter-fn)
+         (take-while not-report)
+         reverse
```

Comment:
double reverse smells. investigate

Reply:
I could be clever and do something that takes results from the end of the list but I suspect `reverse` is faster than I think